pyglmnet/pyglmnet.py

```diff
@@ -352,14 +352,13 @@ def simulate_glm(distr, beta0, beta, X, eta=2.0, random_state=None,
     if not sample:
         return _lmb(distr, beta0, beta, X, eta)
 
-    if random_state is not None:
-        np.random.RandomState(random_state)
+    _random_state = np.random.RandomState(random_state)
```

Comment:
see comment above. `random_state` should be `None` by default

Reply:
What do you mean here? `random_state` has as default `None`, therefore the `RandomState` object won't give deterministic result so the original behaviour should be preserved.

Reply:
okay I thought the numpy `RandomState` container does not accept `None`. But it's good. Looks like it does ...